### PR TITLE
[dbnode] query limits - missed commit with feedback

### PR DESCRIPTION
**Pull request by rallen090**
**What this PR does / why we need it**:

Forgot to push last minor commit on https://github.com/m3db/m3/pull/2627

**Special notes for your reviewer**:

**Does this PR introduce a user-facing and/or backwards incompatible change?**:

```release-note

```

**Does this PR require updating code package or user-facing documentation?**:

```documentation-note

```

### CODE REVIEW

File: src/dbnode/storage/limits/query_limits.go [hunk 1]
```diff
@@ -25,6 +25,7 @@ import (
 	"time"
 
 	"github.com/m3db/m3/src/x/instrument"
+
```
Comment (ryanhall07) [inline src/dbnode/storage/limits/query_limits.go:28]: is this intentional?

**Review by ryanhall07 — APPROVED** [review]

File: src/dbnode/storage/limits/query_limits.go (hunk 1 above)
Comment (robskillington) [reply src/dbnode/storage/limits/query_limits.go:28]: Yes unfortunately the linter is temporarily not running... I've asked a few folks to look into reviving the linter (well.. use a new linter, metalinter we used previously is dead) but no progress yet.

**Review by arnikola — APPROVED** [review]

File: src/dbnode/storage/limits/query_limits.go (hunk 1 above)
Comment (robskillington) [reply src/dbnode/storage/limits/query_limits.go:28]: We order imports:
1. stdlib
2. imports from this repo
3. all other repos

Comment (ryanhall07) [reply src/dbnode/storage/limits/query_limits.go:28]: i'm guessing we want to enable https://github.com/golangci/golangci-lint ?

**Pull request merged**